src/components/Asset/Personalization.tsx

```diff
@@ -26,6 +26,7 @@ export const Personalization = ({ asset }: { asset: Asset }) => {
 					style={{ width: '100%' }}
 				>
 					<div className="flex-grow-1">
+						{/** TODO: chango to 'classNames' for consisten */}
```

Comment:
I notice we made the same behavior we are requesting to change (https://github.com/NordicSemiconductor/asset-tracker-cloud-app-aws-js/pull/260#discussion_r977623543) here as well, but this component is not related to this PR at all. 

Should we update this here as well (for for consistency reasons) or should we just leave it as it is?

Reply:
Not related.

Reply:
Unrelated.